src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToStartHandler.java

```diff
@@ -57,7 +50,7 @@ private void setDirectionTypeDropDown(SscsCaseData sscsCaseData) {
         listOptions.add(new DynamicListItem(APPEAL_TO_PROCEED.toString(), APPEAL_TO_PROCEED.getLabel()));
         listOptions.add(new DynamicListItem(PROVIDE_INFORMATION.toString(), PROVIDE_INFORMATION.getLabel()));
 
-        if (hearingRecordingFeatureEnabled) {
+        if (sscsCaseData.getSscsHearingRecordingCaseData() != null && YesNo.YES.equals(sscsCaseData.getSscsHearingRecordingCaseData().getHearingRecordingRequestOutstanding())) {
```

Comment:
the not null check is not needed, since the second check will test for equality on a String